examples/sentiment/scripts/gpt-neox-20b_peft/gpt-neo-20b_sentiment_peft.py

```diff
@@ -78,13 +82,14 @@ class ScriptArguments:
     model_name=script_args.model_name,
     learning_rate=script_args.learning_rate,
     log_with=script_args.log_with,
-    forward_batch_size=16,
-    batch_size=256,
+    mini_batch_size=script_args.mini_batch_size,
+    batch_size=script_args.batch_size,
+    accelerator_kwargs=dict(gradient_accumulation_steps=script_args.gradient_accumulation_steps),
```

Comment:
I would add `gradient_accumlation_steps` a dedicated kwarg of the `PPOConfig`. The `accelerator_kwargs` should be a bit a power feature in my opinion as you need to know accelerate to know what to pass whereas GA is a pretty standard thing that the PPOTrainer can expose. What do you think?

Reply:
Sounds good. I've updated the PR and rebased on main.